coupler/coupler_types.F90

```diff
@@ -625,6 +783,12 @@ subroutine CT_spawn_1d_2d(var_in, var, idim, jdim, suffix, as_needed)
     if (.not.var_in%set)&
         & call mpp_error(FATAL, trim(error_header) // ' The parent type has not been initialized.')
 
+    ! check only one kind is used
+    if(var_in%set .and. var_in%num_bcs .gt. 0) then
```

Comment:
is the var_in%set needed here since it's checked above?  (not that it significantly matters)

Reply:
nope (at least for the spawn routines), took these out with 47ae5f7